### PR TITLE
[SFS] add 3 files in styles

### PR DESCRIPTION
**Pull request by MaxOvs19**

### CODE REVIEW

**Comment by MaxOvs19** [comment]
[Separate file _variables](https://trello.com/c/rdNYs3kr/5-separate-file-variables)

File: src/styles.scss [hunk 1]
```diff
@@ -3,6 +3,8 @@
 @import url('https://fonts.googleapis.com/css2?family=Forum&family=Montserrat:ital,wght@0,300;0,400;0,500;1,600&display=swap');
 @import './styles/variables';
 @import './styles/typography';
+@import './styles/colors';
+@import './styles/components';
```
Comment (AlexeyOvsyanikov) [inline src/styles.scss:7]: @MaxOvs19 remove unused imports

File: src/styles/_components.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@mixin flex-col-center {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (AlexeyOvsyanikov) [inline src/styles/_components.scss:5]: @MaxOvs19 it's not a 'component'. Layout - possible

File: src/styles/_variables.scss [hunk 3]
```diff
@@ -1,19 +1,5 @@
...
-$forum-font: Forum;
-
-//colors
-$orange: #ffb522;
-$white: #ffffff;
-$ocher: #4f3606;
-$black: rgb(0, 0, 0);
-$violet: #8649ef;
-//shadow
 $base-box-shadow: 0px 4px 28px rgba(0, 0, 0, 0.1);
```
Comment (AlexeyOvsyanikov) [inline src/styles/_variables.scss:1]: @MaxOvs19 i think we can add file '_effects' for shadows etc.

File: src/styles/_variables.scss [hunk 4]
```diff
@@ -1,19 +1,5 @@
...
 $border-routine-table: 0.5px solid #dedede;
 
-//border
-
 //border-menu
 $border-menu: 0.5px solid #dedede;
```
Comment (AlexeyOvsyanikov) [inline src/styles/_variables.scss:5]: @MaxOvs19 Also, if this style uses by one component - don't need to move it in the file. Same as for "$border-routine-table". And think about name "border-routine-table". Is it correct?)

**Pull request merged**